tests/01-ct.sh

```diff
@@ -198,8 +198,35 @@ function connectivity_test() {
 	#}
 }
 
+function cleanup_bpffs() {
+	# Cleanup so we can detect bugs in case these get regenerated when
+	# they shouldn't. We could perhaps let cilium prune stale maps in
+	# future on reconfig?
```

Comment:
Potentially cilium could even do best-effort and migrate existing connection from the global to the local table.